code/script.js

```diff
@@ -1,19 +1,117 @@
 // Start here
 
 // Step 1 - Welcome and introduction
-// Your code goes here
 alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
-)
+);
+
+//Ask user for their name using prompt method (when using conts the variable name doesn't appear in the alert. it says that it's deprecated)
+const userName = prompt(`What's your name? - ex. Maria`);
+
+//Display a greeting to confirm users name
+alert(`Hi ${userName}! Welcome to Javascript Pizzeria.`);
+
 
 // Step 2 - Food choice
-// Your code goes here
+
+const foodChoice = prompt(`Choose the type of food that you'd like to order.
+Enter a number:
+1 - Pizza
+2 - Pasta
+3 - Salad`);
+
+//Why do I already need to define these 2 in step2? Probably this code would look lot cleaner if I had used an array.
+let chosenSubtype; //subtype choice - define dish based on user's choice to alert in next step
+let subtypeMessage; //Prepare for next step because I need 2 variables in the condition to define the correct message 
+
+const foodType = parseInt(foodChoice);
+
+let chosenFoodType; //Declare a variable to store the chosen food type
+
+if (foodType === 1) {
+  chosenFoodType = "Pizza";
+  alert("Wonderful, you've chosen Pizza.");
+} else if (foodType === 2) {
+  chosenFoodType = "Pasta";
+  alert("Wonderful, you've chosen Pasta.");
+} else if (foodType === 3) {
+  chosenFoodType = "Salad";
+  alert("Wonderful, you've chosen Salad.");
+} else {
+  chosenFoodType = alert("Invalid choice. Please select a number between 1-3.");
+}
 
 // Step 3 - Subtype choice
-// Your code goes here
+
+if (chosenFoodType) {
+  let subtype;
+
+  if (chosenFoodType === "Pizza") {
+    subtype = prompt("What type of pizza would you like?\n1 - Margherita\n2 - Pepperoni\n3 - Veggie");
+    subtypeMessage = "Pizza";
```

Comment:
I don't really understand the reason why we need the variable "subtypeMessage" as it has the same value as "chosenFoodType".